src/Umbraco.Cms.Api.Management/Controllers/Package/Created/SaveCreatedPackageController.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Common.Builders;
+using Umbraco.Cms.Api.Management.Factories;
+using Umbraco.Cms.Api.Management.ViewModels.Package;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Packaging;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Api.Management.Controllers.Package.Created;
+
+public class SaveCreatedPackageController : CreatedPackageControllerBase
+{
+    private readonly IPackagingService _packagingService;
+    private readonly IPackageDefinitionFactory _packageDefinitionFactory;
+    private readonly IUmbracoMapper _umbracoMapper;
+
+    public SaveCreatedPackageController(
+        IPackagingService packagingService,
+        IPackageDefinitionFactory packageDefinitionFactory,
+        IUmbracoMapper umbracoMapper)
+    {
+        _packagingService = packagingService;
+        _packageDefinitionFactory = packageDefinitionFactory;
+        _umbracoMapper = umbracoMapper;
+    }
+
+    /// <summary>
+    ///     Creates or updates a package.
+    /// </summary>
+    /// <param name="model">The PackageDefinition model containing the data for a package.</param>
+    /// <returns>The created or updated package.</returns>
+    [HttpPost("save")]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(PackageDefinitionViewModel), StatusCodes.Status200OK)]
+    public async Task<ActionResult<PackageDefinitionViewModel>> Save(PackageDefinitionViewModel model)
```

Comment:
I think this should be two separate endpoints to conform with the new standards, one for explicitly creating a package and one for updating an existing package, the dictionary controllers has examples of this 😄